src/test_helpers.rs

```diff
@@ -0,0 +1,46 @@
+use crate::config::DataContext;
+use std::path::PathBuf;
+use uuid::Uuid;
+
+pub fn create_test_context() -> DataContext {
+    let test_id = Uuid::new_v4().to_string();
```

Comment:
細かいですが Uuid は format できるので to_string() 不要ではないでしょうか

```suggestion
    let test_id = Uuid::new_v4();
```